.github/workflows/is-accl-release-build.yml

```
@@ -0,0 +1,71 @@
+      - name: Locate ZIP file
+        id: locate_zip
+        run: echo "::set-output name=zipfile::$(find ./distribution/is-accelerator/target -name '*.zip' | head -n 1)"
+
+      # Print the ZIP path as a debug step to confirm the artifact was created
+      - name: Print ZIP file path
+        run: echo "ZIP file created at ${{ steps.locate_zip.outputs.zipfile }}"
+
+      # Tag the commit with the extracted version so the release is traceable
+      - name: Create Tag
+        id: create_tag
+        run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)